sarracen/interpolate/interpolate.py

```diff
@@ -770,14 +770,14 @@ def interpolate_2d_line(data: 'SarracenDataFrame',  # noqa: F821
         h_data = data[data.hcol].to_numpy()
 
     grid = get_backend(backend) \
-        .interpolate_2d_cross(data[x].to_numpy(), data[y].to_numpy(),
+        .interpolate_2d_line(data[x].to_numpy(), data[y].to_numpy(),
                               w_data, h_data, kernel.w, kernel.get_radius(),
                               pixels, xlim[0], xlim[1], ylim[0], ylim[1])
 
     if normalize:
         w_norm = _get_weight(data, np.array([1] * len(w_data)), dens_weight)
         norm_grid = get_backend(backend) \
-            .interpolate_2d_cross(data[x].to_numpy(), data[y].to_numpy(),
+            .interpolate_2d_line(data[x].to_numpy(), data[y].to_numpy(),
                                   w_norm, h_data, kernel.w,
                                   kernel.get_radius(), pixels, xlim[0],
                                   xlim[1], ylim[0], ylim[1])
```

Comment:
The linter has picked up that lines 774/775 and 781-783 no longer have indentation that line up with the preceding lines. These need to be shifted left 1 space.